src/Microsoft.DotNet.Build.Tasks/PackageFiles/crossgen.sh

```diff
@@ -85,20 +86,24 @@ __mncaFolder=$__toolsDir/dotnetcli/shared/Microsoft.NETCore.App
 __sharedFxVersion=`ls $__mncaFolder | sed 'r/\([0-9]\+\).*/\1/g' | sort -n | tail -1`
 __sharedFxDir=$__toolsDir/dotnetcli/shared/Microsoft.NETCore.App/$__sharedFxVersion/
 
-case $(uname -s) in
-    Darwin)
-        __packageRid=osx-x64
-        __libraryExtension=dylib
-        ;;
-    Linux)
-        __packageRid=linux-x64
-        __libraryExtension=so
-        ;;
-    *)
-        echo "Unsupported OS $(uname -s) detected. Skipping crossgen of the toolset."
-        exit 0
-        ;;
-esac
+# TODO: once all call-sites start passing <rid> argument to this script
```

Comment:
I would just remove the TODO. It is OK to keep this as a default fallback for anyone using the script that doesn't pass a rid.